docs/includes/nugetsolver-tool.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Include
+description: Suggest NuGetSolver experimental tool.
+author: ErickYondon
+ms.author: eryondon
+ms.date: 1/16/2024
+ms.topic: include
+---
+
+> [!Tip]
+
+> **Alternative solution**: Use NuGetSolver, an automated experimental tool from Microsoft DevLabs(not officially supported by NuGet) for solving dependency conflicts, to help you find solutions to problems like this automatically!
```

Comment:
Great start, I think this is a bit too salesy for docs.

Try to rewrite this more directly/grounded if possible.

NuGetSolver is an experimental Visual Studio Extension from Microsoft DevLabs for solving...

For more information, see NuGetSolver on Visual Studio Marketplace.

Reply:
I edited the text little bit. How about now? https://review.learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1107?branch=pr-en-us-3210